src/content/en/2021/caching.md

```diff
@@ -17,4 +20,435 @@ featured_stat_3: TODO
+- **Cacheability**: Is this content cacheable?
+- **Freshness**: If it is cacheable, how long can it be cached for?
+- **Validation**: If it is cacheable, how do I ensure that my cached version is still fresh?
+
+Headers are meant to be used either alone or together. To determine if the content is **cacheable** and **fresh**, we have:
+- `Expires` specifies an explicit expiration date and time (i.e., when precisely the content expires).
+- `Cache-Control` specifies a cache duration (i.e., how long the content can be cached in the browser relative to when it was requested).
+
+When both are specified, `Cache-Control` takes precedence.
+
+Usage of the `Cache-Control` header has increased steadily since 2019. 74% of responses on Mobile pages included the `Cache-Control` header, while 75% of responses on Desktop pages utilized the header.
+
+{{ figure_markup(
+  image="cache-control-expires.png",
+  caption="Percent of responses that set `Cache-Control` and `Expires` headers.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=816405026&format=interactive",
+  sheets_gid="2102749619",
+  sql_file="header_trends.sql"
+) }}
+
+From 2020, the usage of this specific header increased by 0.71% for Mobile and by 1.13% for Desktop. But on Mobile, we still have 25.1% of requests that use neither `Cache-Control` nor `Expires` headers. This leads us to believe there has been an increase in awareness in the community around proper usage of `Cache-Control`, but we still have a long way to go to full adoption of these headers.
+
+To **validate** the content, we have:
+- `Last-Modified` indicates when the object was last changed. Its value is a date timestamp.
+- `ETag` (Entity Tag) provides a unique identifier for the content as a quoted string. It can take any format the server chooses; it is typically a hash of the file contents, but it could be a timestamp or a simple string.
+
+When both are specified, `ETag` takes precedence.
+
+{{ figure_markup(
+  image="last-modified-etag.png",
+  caption="Percent of responses that set `Last-Modified` and `ETag` headers.",
+  description="TODO",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vSGgVDZ9RkFQLmk5C3_siIcH-8macUEZMobcC0o1z8frYj8NOkI_C2s_yE5ppMdxDAD5INjNsCBa3h1/pubchart?oid=1490687174&format=interactive",
+  sheets_gid="2102749619",
+  sql_file="header_trends.sql"
+) }}
+
+Comparing 2020 with 2021, we notice a similar trend from past years where the `ETag` is becoming slightly more popular, and `Last-Modified` is being used 1.5% less. But what we should probably keep an eye on next year is this new trend in 1.4% more pages using neither `ETag` nor `Last-Modified` headers, as this could imply a challenge in the community understanding the value of these headers.
```

Comment:
Edit: 

Comparing 2020 and 2021, we notice a recurring trend from past years with the `ETag` becoming slightly more popular, and `Last-Modified` being used 1.5% less. What we should probably keep an eye on next year is a new trend of 1.4% more pages using neither `ETag` nor `Last-Modified` headers, as this could imply a challenge in the community understanding the value of these headers.